### DISCUSSION TITLE
Set site Title visibility to on/off using PnP?

### DISCUSSION THREAD

**Discussion by marcvall** [pnp/powershell#212] [Q&A] [answered]
Is there a way to change the site title visibility to On/Off using PowerShell/PnP?

Related to the new SharePoint feature (rolling out) with new site header options (header layouts, visibility, etc.).

**Comment by erwinvanhunen** [comment, marked as answer]
not yet with the built in cmdlets. However, you can do this:

```powershell
$web = Get-PnPWeb
$web.HideTitleInHeader = $false
$web.Update()
Invoke-PnPQuery
```

However, as of tomorrow (2021-02-11, version 1.2.20-nightly) there will be a switch added to Set-PnPWeb


```PowerShell
Set-PnPWeb -HideTitleInHeader # to hide the title
Set-PnPWeb -HideTitleInHeader:$true # to make it visible again.
```

You will have to install the latest nightly to get the updated Set-PnPWeb cmdlet:

```powershell
Install-Module -Name PnP.PowerShell -AllowPrerelease
```

To update to a specific version

```powershell
Install-Module -Name PnP.PowerShell -AllowRerelease -RequiredVersion 1.2.20-nightly
```

**Reply by marcvall** [reply]
Thanks Erwin!

**Comment by AndersRask** [comment]
Don't you mean -HideTitleInHeader:$false   # to make it visible again

🙂

Anders

________________________________

**Reply by ToddKlindt** [reply]
The double negative switches are so confusing. :(
